### PR TITLE
Rewrite docstrings for skll/config/utils.py

### PR DESCRIPTION
**Pull request by RobertImbrie**
- Add space between parameters.
- Remove 'optional' from parameters with default values.
- Move default values next to parameter names.
- Add missing periods to parameter descriptions.
- Add missing double backtick to variable and object names.
- Fix incorrect variable/object reference in _munge_featureset_name docstrings.
    - The docstrings reference `feature_set`, which does not exist. The function is passed a variable called `featureset`. The `FeatureSet` object is spelled with camel case.
    - As it turns out, `feature_set` was used interchangeably for both the `featureset` variable *and* the `FeatureSet` object.
    - `feature_set` reverted to either `featureset` or `FeatureSet`, as appropriate.
- Shortened _munge_featureset_name function description by just a couple words.
    
**Note:** No array parameters this time either!

### CODE REVIEW

File: skll/config/utils.py [hunk 1]
```diff
@@ -110,18 +111,18 @@ def locate_file(file_path, config_dir):
 
 def _munge_featureset_name(featureset):
     """
-    Joins features in featureset by '+' if featureset is not a string, and just
-    returns featureset otherwise.
+    Joins features in ``featureset`` by '+' if ``featureset`` is not a string.
```
Comment (desilinguist) [inline skll/config/utils.py:114]:
```suggestion
    Joins features in ``featureset`` with a '+' if ``featureset`` is not a string.
```

File: skll/config/utils.py [hunk 2]
```diff
@@ -110,18 +111,18 @@ def locate_file(file_path, config_dir):
...
+    Otherwise, returns ``featureset``.
 
     Parameters
     ----------
     featureset : SKLL.FeatureSet
```
Comment (desilinguist) [inline skll/config/utils.py:119]:
```suggestion
    featureset : skll.data.FeatureSet
```

**Review by desilinguist — CHANGES_REQUESTED** [review]
Thanks @RobertImbrie ! A couple of suggestions.

**Comment by RobertImbrie** [comment]
@desilinguist Suggestions added!

**Review by desilinguist — APPROVED** [review]
LGTM!

**Review by mulhod — APPROVED** [review]

**Pull request merged**